src/index.js

```diff
@@ -11,6 +11,8 @@ import rootSaga from './Root/rootSaga';
 import configureStore from './store/configureStore';
 import Root from './Root/Root';
 
+import '../node_modules/grommet/grommet.min.css';
```

Comment:
It means don't import non-js files because that won't work in native environment.

The idea is that js code should continue to work with native platform support without webpack.